### PR TITLE
fix(gas-fee-controller): add missing @metamask/polling-controller

### PR DESCRIPTION
**Pull request by legobeat**
## Explanation

Adds missing dependency introduced in #1673

## References

- Follow-up to: #1673 

## Checklist

- [ ] I've updated the test suite for new or updated code as appropriate
- [ ] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [ ] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

**Review by OGPoyraz — APPROVED** [review]

File: packages/gas-fee-controller/package.json [hunk 1]
```diff
@@ -55,7 +56,8 @@
     "typescript": "~4.8.4"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^13.0.1"
+    "@metamask/network-controller": "^13.0.1",
+    "@metamask/polling-controller": "^0.0.0"
```
Comment (Gudahtt) [inline packages/gas-fee-controller/package.json:60]: Welp. This shouldn't be a peerDependency in this case, but the contraints are enforcing it due to the name.

The intention was to enforce that _controllers_ are peerDependencies of each other. But the `polling-controller` package is not itself a controller, it's a mixin meant to be used by controllers.

Comment (adonesky1) [reply packages/gas-fee-controller/package.json:60]: Not sure how to proceed here @Gudahtt ?

Comment (Gudahtt) [reply packages/gas-fee-controller/package.json:60]: We can add an exception for `polling-controller` to the constraints here: https://github.com/MetaMask/core/blob/a1e1883a376a2e8747fc3e389fb3866b3b663008/constraints.pro#L308

We've already added an exception for two other packages

Comment (adonesky1) [reply packages/gas-fee-controller/package.json:60]: Ok I'll add that in this PR.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**